bitcoin_executor/sources/interpreter.move

```diff
@@ -439,7 +439,7 @@ public fun isSuccess(ip: &Interpreter): bool {
         return false
     };
     let top = ip.stack.top();
-    cast_to_bool(&top)
+    cast_to_bool(&top.destroy_or!(abort EInvalidStackOperation))
```

Comment:
lets add more granual errors, one for pop one for top. Invalid stack operation can be anything